packages/cli/templates/webcomponents/igc-ts/projects/side-nav/files/package.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
@dafo No need to copy the `pacakge.json` (and the 7 other config files in the root alone, haven't checked folders) that are identical between projects, it'd be very cumbersome to maintain multiples.
Move all the shared files in a base (see Angular's setup for working example), put different/additional files in their respective projects.

Reply:
On that note, the current setup looks off to me:
- Index.html imports two modules - index and app. Would be nice to have a single entry point, index (or main) may very well import App internally, right?
- The empty project not having an app file is weird, it's index.html still using `<app-navigation></app-navigation>` in the markup even more so (and it's wrong as it stands, careful!). Like there's still an app as a concept at least. The only difference is the empty proj has basically an outlet being the entire layout.

Reply:
To the last point, the router outlet being external to the app doesn't look quite right, it'll definitely mess with creating more complicated layouts. Is that a limitation of some sorts for the router in use?